code/modules/ruins/spaceruin_code/clericsden.dm

```diff
@@ -0,0 +1,36 @@
+///////////	cleric's den items.
+
+//Primary reward: the cleric's mace design disk.
+/obj/item/disk/design_disk/adv/cleric_mace
+	name = "Enshrined Disc of Smiting"
+
+/obj/item/disk/design_disk/adv/cleric_mace/Initialize()
+	. = ..()
+	var/datum/design/cleric_mace/M = new
+	blueprints[1] = M
+
+/obj/item/paper/fluff/ruins/clericsden/contact
+	info = "Father Aurellion, the ritual is complete, and soon our brothers at the bastion will see the error of our ways. After all, a god of clockwork or blood? Preposterous. Only the TRUE GOD should have so much power. Signed, Father Odivallus."
+
+/obj/item/paper/fluff/ruins/clericsden/warning
+	info = "FATHER ODIVALLUS DO NOT GO FORWARD WITH THE RITUAL. THE ASTEROID WE'RE ANCHORED TO IS UNSTABLE, YOU WILL DESTROY THE STATION. I HOPE THIS REACHES YOU IN TIME. FATHER AURELLION."
+
+/mob/living/simple_animal/hostile/construct/proteon
+	name = "Proteon"
+	real_name = "Proteon"
+	desc = "A weaker construct meant to scour ruins for objects of Nar'Sie's affection. Those barbed claws are no joke."
```

Comment:
wait, doesn't one of the fluff messages mention that this monastery ISN'T dedicated to nar-sie? why are Her constructs here?

Reply:
It's implied that whatever holy ritual they performed backfired, summoning or attracting constructs to the monastery, and destroying the attached station.